src/app/visual/filter/filter-visual.component.html

```diff
@@ -1,3 +1,10 @@
+<sky-docs-demo-control-panel-theme
+  (themeSettingsChange)="themeSettingsChange($event)"
+></sky-docs-demo-control-panel-theme>
+
+<br/>
+
+<div [skyTheme]="themeSettings">
```

Comment:
You don't need the `skyTheme` directive. The global `SkyThemeService` that you're injecting in the constructor will handle it.